docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,67 @@
+the sync status and detect any issues with the offset syncing process.
+Run the tool using the following command:
+
+```bash
+<extracted tgz>/bin/kafka-mirrormaker-offset-sync-inspector.sh --mm2-config <path-to-config-file>
+```
+
+Parameter:
+
+- `--mm2-config <path-to-config-file>`: Specifies the path to the Aiven for Apache Kafka
+  MirrorMaker 2 configuration file. This file contains the connection details for the
+  source and target Apache Kafka clusters.
+
+To view all available options, run the help command:
+
+```bash
+./bin/kafka-mirrormaker-offset-sync-inspector.sh --help
+```
+
+## Log messages and actions
+
+Below are common log messages generated by the tool, what they mean, and recommended
+actions. If any issues arise, [create a support ticket](/docs/platform/howto/support) or
+email [Aiven Support](mailto:support@aiven.io).
+
+| Sync status message | Description | Action required | Inspect | Analyze |
+|---------------------|-------------|-----------------|---------|---------|
+| Successfully synced, operating normally. | Data and offsets are successfully migrated, allowing consumers to read from synced offsets with minimal redelivery. | No action required. | None. | None. |
+| Intentionally not synced because the source partition is empty. Other partitions in this group are synced. | The source partition has no data, so Apache Kafka MirrorMaker 2 skips syncing it. Consumers start from the beginning when new data arrives. | No action required. | None. | None. |
+| Intentionally not synced because the source partition is empty. Other partitions in this group are not synced. | The source partition has no data, so Apache Kafka MirrorMaker 2 skips syncing it. Consumers start from the beginning when new data arrives. | No action required. | None. | None. |
+| Intentionally not synced because the source group is too old. Other partitions in this group are synced. | The source partition has newer data than the committed offset for the consumer group. Consumers start from the beginning when new data arrives. | No action required. | None. | None. |
+| Erroneously not synced. Source partition has data, and other partitions for this group are synced. | There is an offset syncing issue. Consumers start from the beginning when new data arrives. | [Create a support ticket](/docs/platform/howto/support). | - Use `kafka-get-offsets` to compare the earliest and latest offsets in source and target topics.<br /> - Check the source partition's throughput on the Apache Kafka dashboard.<br /> - Inspect offset syncs in `kafka-console-consumer` to locate the topic-partition key.<br /> - Enable TRACE logging for OffsetSyncStore in Apache Kafka MirrorMaker 2. | - The partition has little or no data, causing dropped syncs.<br /> - Zero throughput prevents syncs from being generated.<br /> - Verify if MirrorSourceTask isn’t emitting syncs or if MirrorCheckpointTask is expiring them. |
```

Comment:
I also ran into the issue with bullet points looking different in tables in another doc and ended up using list tags `<li>`.

Reply:
Thanks for this suggestion :raised_hands: